### PR TITLE
add ncbi-datasets to the knowledgebase

### PR DESCRIPTION
**Pull request by huzuner**

### CODE REVIEW

**Comment by dlaehnemann** [comment]
Yeah, it's always tricky to decide where to put stuff. I'd probably start a new folder `src/data_downloads/` and name the file `src/data_downloads/ncbi_datasets_cli.md`.

Also, this would require an update to `src/SUMMARY.md` to have table of content entry for this new category. And we could also create a new article about the ensembl FTP resources (and the snakemake wrapper(s) that can use it). But the latter, I'd do in a new PR.

File: src/SUMMARY.md [hunk 1]
```diff
@@ -23,6 +23,9 @@
 - [Data analyses](data_analyses/README.md)
   - [differential expression analysis](data_analyses/differential_expression.md)
 
+- [Data downloads](data_downloads/README.md)
+  - [ncbi_datasets](data_downloads/ncbi-datasets-cli.md)
```
Comment (dlaehnemann) [inline src/SUMMARY.md:27]: Sorry to be nit-picking, but I'd actually go for underscores, here. I try to stick to [`snake_case` for naming things](https://r4ds.hadley.nz/workflow-basics.html#sec-whats-in-a-name), for readability and consistency. So:
```suggestion
  - [ncbi_datasets](data_downloads/ncbi_datasets_cli.md)
```

Comment (huzuner) [reply src/SUMMARY.md:27]: You're definitely right, I too almost always use underscores but sometimes something just goes with the flow of your fingers...

File: src/data_downloads/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+In this section, tools for downloading data that may include raw sequencing data, genes, genomes, annotations.. etc. are gathered.
```
Comment (dlaehnemann) [inline src/data_downloads/README.md:1]:
```suggestion
In this section, tools for downloading data that may include raw sequencing data, genes, genomes, annotations, etc. are gathered.
```

File: src/data_downloads/ncbi_datasets_cli.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# ncbi-datasets: gathering data across NCBI databases
+
+[ncbi-datasets](https://github.com/ncbi/datasets) is a very-well documented open source tool from NCBI (National Center for Biotechnology Information) for gathering various kinds of data including genes, genomes, annotations etc. belonging to the desired species.
+As a side note, it has specific options for retrieving SARS-CoV-2 data.
+
+## Installation
+
+It is offered as a conda package and can be downloaded via:
```
Comment (dlaehnemann) [inline src/data_downloads/ncbi_datasets_cli.md:8]:
```suggestion
It is offered as a conda package and can be installed via:
```

File: src/data_downloads/ncbi_datasets_cli.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+```
+conda install -c conda-forge ncbi-datasets-cli
```
Comment (dlaehnemann) [inline src/data_downloads/ncbi_datasets_cli.md:11]: I usually try to work with isolated and minimal conda environments, so I'd suggest to also make this the default here:
```suggestion
conda create -n ncbi-download -c conda-forge ncbi-datasets-cli
```

File: src/data_downloads/ncbi_datasets_cli.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+## Example usage
```
Comment (dlaehnemann) [inline src/data_downloads/ncbi_datasets_cli.md:14]: Great idea, to add some good examples right here!

**Review by dlaehnemann — APPROVED** [review]
Thanks a bunch. Just some small suggestions, feel free to accept or decline.

**Comment by huzuner** [comment]
Thank you for your comments!

**Pull request merged**